src/pages/node-express/lesson6-m1.mdx

```diff
@@ -0,0 +1,30 @@
+---
+layout: "../../layouts/genericMarkdownFile.astro"
+title: "Node/Express Lesson 6: Task Manager API Part 2"
+description: Material link for lesson 6 of the node/express class
+---
+
+import {
+  AssignmentLinkButton,
+  NextLinkButton,
+  PrevLinkButton,
+} from "../../components/navButtons.js";
+
+# Node/Express Lesson 6
+
+# Task Manager API Part 2
+
+This week begins at 1:29 of **[this video](https://youtu.be/rltfdjcXjmk?t=5322)**, and continues until the end of the task manager project, at 3:07:18 of the video.
```

Comment:
updated timestamp to 5322 seconds (equivalentish to 1hr 29 minutes called out in the text)